packages/quantic/cypress/integration/quantic-facet.cypress.ts

```diff
@@ -31,4 +65,103 @@ describe('quantic-facet', () => {
       .get(facet.less)
       .should('not.exist');
   });
+
+  it('should collapse and expand when clicking the button', () => {
+    configure({
+      numberOfValues: 8,
+      noSearch: false,
+    })
+      .get(facet.collapse)
+      .click()
+      .get(facet.title)
+      .should('exist')
+      .get(facet.value)
+      .should('not.exist')
+      .get(facet.more)
+      .should('not.exist')
+      .get(facet.searchbox)
+      .should('not.exist')
+      .get(facet.expand)
+      .click()
+      .get(facet.title)
+      .should('exist')
+      .get(facet.searchbox)
+      .should('exist')
+      .get(facet.value)
+      .should('have.length', 8)
+      .get(facet.more)
+      .should('exist');
+  });
+
+  it('should search when typing in the searchbox', () => {
+    configure({
+      noSearch: false,
+    })
+      .get(facet.value)
+      .first()
+      .then(($value) => {
+        const firstFacetValue = $value.attr('data-cy') ?? '';
+
+        cy.get(facet.searchbox)
+          .type(firstFacetValue)
+          .get(facet.value)
+          .should('have.length', 1)
+          .get(facet.searchClear)
+          .click()
+          .get(facet.value)
+          .should('have.length', 8);
+      });
+  });
+
+  it('should filter results when selecting values', () => {
+    setupSearchAlias()
+      .then(() => configure({}))
```

Comment:
Do you think it would be worthwhile to make `{}` the default parameter for the configure method ?
(It is supported https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)

Reply:
Yes, I also think it would make sense.